src/onegov/org/cli.py

```diff
@@ -884,3 +886,54 @@ def mark_as_published(request: 'OrgRequest', app: 'OrgApp') -> None:
             )
 
     return mark_as_published
+
+
+@cli.command(name="delete-invisible-links")
+def delete_invisible_links() -> 'Callable[[OrgRequest, OrgApp], None]':
+    """ Deletes all the data associated with a period, including:
+    Example:
+        onegov-org --select /foo/bar delete-invisible-links
+    """
+
+    def delete_invisible_links(request: 'OrgRequest', app: 'OrgApp') -> None:
+        session = request.session
+        pages = session.query(Page).all()
```

Comment:
Technically you would need to do this for any model that uses the `GeneralFileLinkExtension`, not just `Page`, if you wanted to be thorough.